File: JavaNgArch/src/projectbase/domain/NamedIntEntity.java
package projectbase.domain;

import projectbase.sharparch.domain.domainmodel.DomainSignature;

public class NamedIntEntity extends BaseDomainObject {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String _name;

	@DomainSignature
	public String getName() {
		return _name;
	}

	public void setName(String value) {
		this._name = value;
	}

}

File: NetNgArch/src/ProjectBase/Domain/NamedIntEntity.cs
using System;
using SharpArch.Domain.DomainModel;

namespace ProjectBase.Domain
{
    [Serializable]
    public class NamedIntEntity : BaseDomainObject
    {
        #region "persistent properties"

        [DomainSignature]
        public virtual String Name { get; set; }

        #endregion

        #region "extended properties"

        #endregion

        #region "methods"


        #endregion
    }
}
